### ISSUE TITLE
Design review for grid

### ISSUE THREAD

**Issue by LauraHilliger** [mozilla/webmaker-curriculum#50]
Hey @iamjessklein - any chance you could take a look and give some feedback on:

http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session01-credibility.html

I tried to follow your design, but also used CSS Flexbox - so make your screen bigger smaller. Also, perhaps @toolness can get involved here and take a quick peek at the html/css?

**Comment by toolness** [comment]
Aww yeah flexbox **ROCKS**!!

**Comment by toolness** [comment]
I took a very quick look at the HTML and CSS and it looks nice. The only visual bug I noticed is that the "Pledge" button appears to overflow its container on narrow/mobile displays:

![screen shot 2015-01-07 at 1 48 53 pm](https://cloud.githubusercontent.com/assets/124687/5651267/ff189b34-9673-11e4-8b09-565b893a4cad.png)

**Comment by LauraHilliger** [comment]
@toolness - do you have an idea for an easy fix there? I haven't written a mobile.css, but maybe if that's the only hiccup we don't need to...

**Comment by keyboardkat** [comment]
This is a really awesome design, @LauraHilliger. I love it! 

i'd suggest leaving a slot or two in for additional images on the bottom left-hand of the window, in the grey area that's currently blank (in case folks have thumbnails or pics from events they'd like to slot in). 

Also, where can you find the raw code for this? Feel like I'm missing a link somewhere!

**Comment by smithisgeneric** [comment]
I assume the Web Literacy Competencies section will act as a menu to access other related activities?  If there are multiple activities under a single competency this section will need some design work.

**Comment by LauraHilliger** [comment]
@keyboardkat code for the first module is here: https://github.com/mozilla/webmaker-curriculum/tree/master/WebLiteracyBasics-I

If you have a couple image sugggestions, I'll build them in now. Then we can copy/paste this session's code for all the other ones.

For now, we're doing pretty rudimentary html/css – once the content is finished, we'll likely need Product to help us build the infrastructure for presenting it all. In the meantime, strap on your a tags!

**Comment by keyboardkat** [comment]
re photos on lefthand side: perhaps a section entitled "Activity Photos" or something like it, with: 

* https://farm9.staticflickr.com/8129/8707135735_fdacf1a0a4_b.jpg "Groupwork in action with the Mozilla Indonesia community."
* https://farm4.staticflickr.com/3759/12030099454_7f80baaa4b_b.jpg "Groups break into teams at a Hive Chicago maker party"

Ideally we'd have pictures to use of people using the Kraken print-outs, ala the [Design Canvas](http://mzl.la/1zpk696) activity, but maybe that could work for example version before it's hard-coded?

**Comment by LauraHilliger** [comment]
Ok, I've added an image section, and we can replace the photos with people doing the Kraken activity later. @iamjessklein @keyboardkat @dajbelshaw have anymore comments on layout for this page or should we could close this issue?

http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session01-credibility.html

**Comment by dajbelshaw** [comment]
The mobile view is *slightly* wonky (some divs not in alignment and buttons in odd places) but for testing it's an R+ from me. The desktop view is what people will be using most and it's looking gooooood.

**Comment by iamjessklein** [comment]
My main concern is the few glitches in terms of responsiveness (the pledge button etc) and I haven't checked how this looks in browsers other than Firefox, but I would take a look at that. Otherwise for this testing I would say ship it. :shipit:

**Comment by LauraHilliger** [comment]
agreed. mobile is janky, but for session 1 testing. F it, shipping. Opening a new issue for mobile optimization (for which I'd like a *real* dev ;)

**Comment by keyboardkat** [comment]
mmmm, I'm LOVIN' it. looks awsm w photos. shall move to mobile thread.
